src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml

```diff
@@ -105,7 +105,7 @@
       IsReadOnly="{Binding IsProjectPackageReference, Converter={StaticResource NotNullOrTrueToBooleanConverter}}"
       StaysOpenOnEdit="{Binding IsProjectPackageReference, Converter={StaticResource InverseNullToVisibilityConverter}}"
       ItemsSource="{Binding Path=VersionsView}"
```

Comment:
Should ItemsSource Binding be TwoWay? My is that If drop down items are not showing, then the underlying collection is not changing.